src/neptune/exceptions.py

```diff
@@ -824,25 +824,17 @@ def __init__(self, reason: str):
 {end}
 {reason}
 
-Data synchronization was interrupted because you're out of storage space (or logging hours). If you're using Neptune in
-asynchronous mode (default), the data that couldn't be uploaded is safely stored on the disk.
-You can still fetch and delete data from your project(s).
-How can I free up space in Neptune?
-    - Contact your workspace admin about possible project or workspace storage limits.
-    - Upgrade your subscription with more storage: https://app.neptune.ai/-/subscription
-    - Delete run or model metadata that you don't need.
+Some limit related to your workspace or project was exceeded. You can manage your plan and check usage in the workspace
+settings: https://app.neptune.ai/-/subscription
```

Comment:
@PatrykGala Is it possible to inject the exact workspace into the link? If the user is a member of multiple workspaces, this link won't necessarily take them to the correct one.

Reply:
I'll add it in a separate PR, so that it would not stop this PR from being merged 😇